cli/auth.go

```diff
@@ -156,12 +156,13 @@ func (c *viamClient) loginAction(cCtx *cli.Context) error {
 	if currentToken != nil && currentToken.canRefresh() {
 		t, err = c.authFlow.refreshToken(c.c.Context, currentToken)
 		if err != nil {
-			utils.UncheckedError(c.logout())
-			return err
+			utils.UncheckedError(c.logout()) // clear cache if failed to refresh
+			return errors.New("error while refreshing token, logging out. Please log in again.")
```

Comment:
@zaporter-work you're right, I missed it in the first pass - I think the error was emitted here, so change it as well